integration-tests/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id 'groovy'
+    id 'java'
+    id 'dwh-migration-dumper.java-library-conventions'
+}
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+    maven {
+        url "https://redshift-maven-repository.s3-website-us-east-1.amazonaws.com/release"
+    }
+}
+
+dependencies {
+    implementation 'org.codehaus.groovy:groovy-all:3.0.5'
+    implementation 'org.testng:testng:7.5'
```

Comment:
Does that limit the scope of lib usage as it does `<scope>test</scope>` maven pom.xml option?

Reply:
yes, correct

Reply:
Then we don't want it, because we use TestNG annotations in Test scope like `@Test` or `@BeforeClass`.
So, because of that I used just `implementation` here.